# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../babytrans/compile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- babytrans-0.9.1/src/main.cc.vanilla	2006-10-05 10:47:43.000000000 +0000
+++ babytrans-0.9.1/src/main.cc	2006-10-05 10:47:59.000000000 +0000
@@ -854,7 +854,7 @@
         }
 
         string              human_attrib ;
-        assert( it->attrib.size() >= 8 ) ;
+        g_assert( it->attrib.size() >= 8 ) ;
         if ( it->attrib[ 2 ] > '1'
              || it->attrib[ 3 ] != '0' ) {
             /* noun */
